projects/packages/jetpack-mu-wpcom/src/features/wpcom-comments/wpcom-comment-like.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+const displayNotice = message => {
+	const headerEnd = document.querySelector( '.wp-header-end' );
+	if ( ! headerEnd ) {
+		return;
+	}
+
+	const dismissText = window.wpcomCommentLikesData.dismiss_notice_text;
+
+	// Create notice container.
+	const notice = document.createElement( 'div' );
+	notice.className = 'notice notice-error is-dismissible';
+	// Add role="alert" so that screen readers announce the notice.
+	notice.setAttribute( 'role', 'alert' );
+
+	// Create paragraph element for the message.
+	const p = document.createElement( 'p' );
+	p.innerText = message;
+	notice.appendChild( p );
+
+	// Create dismiss (close) button.
+	const button = document.createElement( 'button' );
+	button.setAttribute( 'type', 'button' );
+	button.className = 'notice-dismiss';
+	button.setAttribute( 'aria-label', dismissText );
+
+	// Create a span element for screen readers.
+	const span = document.createElement( 'span' );
+	span.className = 'screen-reader-text';
+	span.innerText = dismissText;
+	button.appendChild( span );
+
+	// Hook up the dismiss functionality.
+	button.addEventListener( 'click', () => {
+		notice.parentNode.removeChild( notice );
+	} );
+
+	// Append the button to the notice.
+	notice.appendChild( button );
+
+	// Insert the notice after the headerEnd element.
+	if ( headerEnd.nextSibling ) {
+		headerEnd.parentNode.insertBefore( notice, headerEnd.nextSibling );
+	} else {
+		headerEnd.parentNode.appendChild( notice );
+	}
```

Comment:
Like `Element#remove`, [`Element#after`](https://developer.mozilla.org/en-US/docs/Web/API/Element/after#syntax) is marked "widely available":

```suggestion
	headerEnd.after( notice );
```

Reply:
Updated